artiq/coredevice/phaser.py

```diff
@@ -549,6 +568,50 @@ def get_dac_temperature(self) -> TInt32:
         """
         return self.dac_read(0x06, div=257) >> 8
 
+    @kernel
+    def dac_sync(self):
+        """Trigger DAC synchronisation for both output channels.
+        If the DAC-NCO is enabled, this applies NCO frequency changes."""
+        config_1f = self.dac_read(0x1f)
+        delay(.1*ms)
+        self.dac_write(0x1f, config_1f | (1 << 1))
+        self.dac_write(0x1f, config_1f & ~int32(1 << 1))
+
+    @kernel
+    def set_dac_cmix(self, fs_8_step):
+        """Set the DAC coarse mixer frequency for both channels
+
+        The selected coarse mixer frequency becomes active without explicit
+        synchronisation.
+
+        Use of the coarse mixer requires the DAC mixer to be enabled. The mixer
+        can be configured via the `dac` device_db entries.
```

Comment:
Do you mean `... must be enabled via..`? And the fact that those may come from the device_db is a technical detail that is not guaranteed. You can also instantiate Phaser in different ways. I'd remove that in favor of saying "must be enabled through the `dac` configuration dictionary (see init())`.

Reply:
> "must be enabled through the `dac` configuration dictionary (see init())`.

Ack.

The the mixer can also be enabled by manually using `dac_write`. The device_db comment is intended to point unfamiliar users towards an appropriate place to configure this.

Reply:
Then we should rather do that (tell people that the initial settings dictionaries are accessible through the device db) summarily in the class or __init__ docstring and not in every method.